src/status_im/contexts/wallet/networks/config.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im.contexts.wallet.networks.config
```

Comment:
now, when adding a new chain, most of the configuration would happen here (except for adding the network image to quo resources)